paper-dropdown.js

```diff
@@ -422,7 +454,7 @@ class PaperDropDown extends Polymer.mixinBehaviors([PaperDropdownBehavior], Poly
         if (this.multi) {
             this.set('value', selectedItems.map(item => this._getItemValue(item)));
 
-            const { items } = this.$.list;
+            const {items} = this.$.list;
```

Comment:
We are going to stick to airbnb code style from now on. This change will cause lint issues.
You must have got error which trying to commit, please don't ignore lint error message as soon I will add travis build process to have all these checks.